modules/ROOT/partials/import-database.adoc

```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j 5, replacing the `push-to-cloud` command that was present in Neo4j v4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
```

Comment:
```suggestion
If you have an earlier version of Neo4j, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
```

Reply:
Sorry, I missed this one before. Again, the tool itself is not versioned independently of Neo4j, so we should try not to talk about the version of the tool, but of Neo4j.